packages/analytics-js-integrations/__tests__/integrations/Adroll/browser.test.js

```
@@ -0,0 +1,235 @@
+  window.__adroll = {};
+  // Add a dummy script as it is required by the init script
+  const scriptElement = document.createElement('script');
+  scriptElement.type = 'text/javascript';
+  scriptElement.id = 'dummyScript';
+  const headElements = document.getElementsByTagName('head');
+  headElements[0].insertBefore(scriptElement, headElements[0].firstChild);
+});
+afterEach(() => {
+  // Reset DOM to original state
+  document.getElementById('dummyScript')?.remove();
+  delete window.__adroll;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)